tests/auth_tests/test_models.py

```diff
@@ -176,6 +176,24 @@ def test_last_login_default(self):
         user2 = User.objects.create_user(username='user2')
         self.assertIsNone(user2.last_login)
 
+    def test_user_save_normalize_email(self):
+        """
+        Calling user.save() will call normalize_email()
+        """
+        with mock.patch('django.contrib.auth.models.UserManager.normalize_email') as normalize_email:
```

Comment:
Ok so you want to check the final result of normalization after cleaning model as well ?